content/questions/admin/question-044.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "Which GitHub plans allow organizations to own an unlimited number of public and private repositories with a full feature set on private repositories? (choose two.)"
```

Comment:
What is "full feature set" ? 

GitHub Team for example does not include GitHub Advanced Security features for private repositories.

Make this more clear please + a link to what this refers to